.gitlab-ci.yml

```diff
@@ -224,6 +226,22 @@ API:
         INTERNAL_NETWORK: ["true"]
         DISTRO_CODE: ["rhel_90"]
 
+Installer:
+  stage: test
+  extends: .terraform
+  rules:
+    - if: '$CI_PIPELINE_SOURCE != "schedule"'
+    - if: '$CI_PIPELINE_SOURCE == "schedule" && $RUNNER =~ /[\S]+rhel-[8-9]\.[\S]+/'
+  variables:
+    EXTRA_REPO_PATH_SEGMENT: "gitlab/"
```

Comment:
This has been removed by @ondrejbudai in #1510. You can drop the EXTRA_REPO_PATH_SEGMENT variable.

Reply:
Removed.  Thanks.